libmozdata/connection.py

```diff
@@ -105,6 +109,8 @@ def cb(sess, res):
                     query.handler(response, query.handlerdata)
                 else:
                     query.handler(response)
+            elif self.RAISE_ERROR:
+                raise res.raise_for_status()
```

Comment:
I think `raise` is not needed, raise_for_status will raise itself.

Reply:
you're right: I just forgot to remove it